changelogs/fragments/licensing.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - Use the correct GPL license for plugin_utils (https://github.com/ansible-collections/community.hashi_vault/issues/276).
+  - Add missing ``simplified_bsd.txt`` file for BSD licensed content (https://github.com/ansible-collections/community.hashi_vault/issues/275).
```

Comment:
Nit: The actual PR has `LICENSES/BSD-2-Clause.txt`

Reply:
this was just a lag in rebasing, it's updated now in https://github.com/ansible-collections/community.hashi_vault/pull/282/commits/126b9e6581e2055fd6e099cc721e365b67507ee5 :)

Reply:
@gotmax23 should be all set now if you want to take a look